### DISCUSSION TITLE
Spanish (ES) Lozalization

### DISCUSSION THREAD

**Discussion by dontic** [heyform/heyform#137] [Ideas]
[I've forked the project](https://github.com/dontic/heyform) and started localizing the app in Spanish, more specifically Spanish (Spain).

Contribution docs say to open an issue first. Let me know if I should follow any specific steps.

I'm just starting by creating an `es.ts` file in all 3 `locales` directories:

- packages/webapp/src/locales/es.ts
- packages/form-renderer/src/locales/es.ts
- packages/webapp/src/pages/form/views/FormComponents/locales/es.ts

**Comment by iMuFeng** [comment]
We are migrating `webapp/src/pages/form/views/FormComponents` to `form-renderer` as an npm package. Therefore, only need to add translation files in `FormComponents`. We will merge these into `form-renderer` later.
